morphocluster/frontend/src/views/Project.vue

```diff
@@ -1,82 +1,53 @@
-                    </li>
-                    <li class="nav-item">
-                        <router-link class="nav-link"
-                            :to="{ name: 'project', params: { project_id: project.project_id } }">{{
-                                project.name }}</router-link>
-                    </li>
-                </ul>
-            </div>
-            <dark-mode-control />
-        </nav>
-        <div class="container">
-            <table id="table" style="width=100%">
-                <tbody>
-                    <tr>
-                        <td>Created on</td>
-                        <td>{{ project.creation_date }}</td>
-                    </tr>
-                    <tr>
-                        <td>Name</td>
-                        <td>{{ project.name }}</td>
-                    </tr>
-                    <tr>
-                        <td>Node_id</td>
-                        <td>{{ project.node_id }}</td>
-                    </tr>
-                    <tr>
-                        <td>Project_id</td>
-                        <td>{{ project.project_id }}</td>
-                    </tr>
-                    <tr>
-                        <td>Visible</td>
-                        <td>{{ project.visible }}</td>
-                    </tr>
-                </tbody>
-            </table>
-            <div style="margin: auto ; width: 0; padding-top: 7px ">
-                <b-button size="sm" variant="primary" href="" @click.prevent="showSaveModal(project)">
-                    Save Project
-                </b-button>
-            </div>
-        </div>
-        <b-modal ref="saveModal" lazy centered :title="save_title" @ok="HandleSaveOk">
-            <div class="d-block text-center">
-                <form @submit.stop.prevent="HandleSaveOk">
-                    <b-container fluid>
-                        <b-form-row class="my-1">
-                            <b-col sm="3">
-                                <label for="form_save_name">Name:</label>
-                            </b-col>
-                            <b-col sm="9">
-                                <b-form-input type="text" placeholder="Enter a name for the saved file" id="form_save_name"
-                                    v-model="save_slug"></b-form-input>
-                            </b-col>
-                        </b-form-row>
-                        <b-row class="my-3" v-if="save_saving">
-                            <b-col sm="12"> Saving... </b-col>
-                        </b-row>
-                    </b-container>
-                </form>
-            </div>
-        </b-modal>
+        <v-container>
+            <v-card>
+                <v-card-title>Projektinformationen</v-card-title>
+                <v-card-text>
+                    <v-row v-for="(value, key) in project" :key="key">
+                        <v-col>{{ key }}</v-col>
+                        <v-col>{{ value }}</v-col>
+                    </v-row>
```

Comment:
Use a regular, hard-coded table. When using `v-row` and `v-col`, both columns have the same width. This is ugly. Also, we want human-readable row headers.